### PR TITLE
Improves the docstring of rigid body 

### LINKED ISSUE #24451: sympy.physics.mechanics // RigidBody
I looked at the 'explanations' of RigidBody, using _help(me.RigidBody)._

For the kinetic energy I saw this:

_'T = 1/2 (I omega^2 + m v^2)'_

Should it not be:

T = 1/2 ( **(** I omega **)**^2 + m v^2 )

A **very minor** point, but since I saw it, I wanted to point it out.

### PR DESCRIPTION
**Pull request by 1e9abhi1e10**
#### References to other Issues or PRs

Fixes #24451 

#### Release Notes

NO ENTRY

### CODE REVIEW

File: sympy/physics/mechanics/rigidbody.py [hunk 1]
```diff
@@ -182,7 +182,7 @@ def angular_momentum(self, point, frame):
         The angular momentum H of a rigid body B about some point O in a frame
         N is given by:
 
-            H = I . w + r x Mv
+            `H = dot(I, w) + cross(r, M) * v`
```
Comment (ThePauliPrinciple) [inline sympy/physics/mechanics/rigidbody.py:185]: Should it not be `cross(r, Mv)`? I think cross products are only defined on vectors.

Comment (1e9abhi1e10) [reply sympy/physics/mechanics/rigidbody.py]: Sorry by mistake, I corrected it

File: sympy/physics/mechanics/particle.py [hunk 2]
```diff
@@ -124,7 +124,7 @@ def angular_momentum(self, point, frame):
         The angular momentum H, about some point O of a particle, P, is given
         by:
```
Comment (TJStienstra) [inline sympy/physics/mechanics/particle.py:125]: Maybe another improvement for the sake of consistency is either have a `:` after every `given by` or remove it everywhere.

File: sympy/physics/mechanics/rigidbody.py [hunk 3]
```diff
@@ -182,7 +182,7 @@ def angular_momentum(self, point, frame):
         The angular momentum H of a rigid body B about some point O in a frame
         N is given by:
 
-            H = I . w + r x Mv
+            `H = dot(I, w) + cross(r, M*v)`
```
Comment (TJStienstra) [inline sympy/physics/mechanics/rigidbody.py:185]: Could you make the indentations of this equation equal with respect to the other docstring equations? What I mean is the consistent usage of one of the following in each of these docstrings:
```py
        `equation`
```
or
```py
            `equation`
```

File: sympy/physics/mechanics/particle.py [hunk 4]
```diff
@@ -124,7 +124,7 @@ def angular_momentum(self, point, frame):
...
 
-        H = r x m * v
+        `H = cross(r, m*v)`
```
Comment (TJStienstra) [inline sympy/physics/mechanics/particle.py:127]: Might be nice to use pep8 formatting in the equations, so `H = cross(r, m * v)`

**Review by tjstienstra — COMMENTED** [review]
Thanks for taking the time to update the documentation.

File: sympy/physics/mechanics/particle.py [hunk 5]
```diff
@@ -124,7 +124,7 @@ def angular_momentum(self, point, frame):
         The angular momentum H, about some point O of a particle, P, is given
         by:
 
-        H = r x m * v
+        `H = cross(r, m * v)`
```
Comment (sylee957) [inline sympy/physics/mechanics/particle.py:127]:
```suggestion
        ``H = cross(r, m * v)``
```

File: sympy/physics/mechanics/particle.py [hunk 6]
```diff
@@ -165,9 +165,9 @@ def kinetic_energy(self, frame):
         Explanation
         ===========
 
-        The kinetic energy, T, of a particle, P, is given by
+        The kinetic energy, T, of a particle, P, is given by:
 
-        'T = 1/2 m v^2'
+        `T = 1/2 (dot(m * v, v))`
```
Comment (sylee957) [inline sympy/physics/mechanics/particle.py:170]:
```suggestion
        ``T = 1/2 (dot(m * v, v))``
```

File: sympy/physics/mechanics/rigidbody.py [hunk 7]
```diff
@@ -182,7 +182,7 @@ def angular_momentum(self, point, frame):
         The angular momentum H of a rigid body B about some point O in a frame
         N is given by:
 
-            H = I . w + r x Mv
+        `H = dot(I, w) + cross(r, M * v)`
```
Comment (sylee957) [inline sympy/physics/mechanics/rigidbody.py:185]:
```suggestion
        ``H = dot(I, w) + cross(r, M * v)``
```

File: sympy/physics/mechanics/rigidbody.py [hunk 8]
```diff
@@ -230,9 +230,9 @@ def kinetic_energy(self, frame):
         Explanation
         ===========
 
-        The kinetic energy, T, of a rigid body, B, is given by
+        The kinetic energy, T, of a rigid body, B, is given by:
 
-        'T = 1/2 (I omega^2 + m v^2)'
+        `T = 1/2 * (dot(dot(I, w), w) + dot(m * v, v))`
```
Comment (sylee957) [inline sympy/physics/mechanics/rigidbody.py:235]:
```suggestion
        $T = 1/2 * (dot(dot(I, w), w) + dot(m * v, v))$
```

**Review by sylee957 — COMMENTED** [review]
You need to check carefully not to use single backticks, 
but to use double backticks or dollars depending on whether the formula is in plaintext or LaTeX.
You can check how the actual output looks like
https://output.circle-artifacts.com/output/job/e371edf3-5831-4ece-a5dc-50ccbf1cedf6/artifacts/0/doc/_build/html/modules/physics/mechanics/api/part_bod.html#sympy.physics.mechanics.particle.Particle.angular_momentum

**Comment by sylee957** [comment]
You need to follow the instructions to add the authors
https://github.com/sympy/sympy/actions/runs/3884223548/jobs/6626539348
This only needs to be done for the first commit

**Comment by sylee957** [comment]
The mailmap issue still needs to be resolved.

**Pull request merged**